lte/gateway/c/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -577,4 +576,5 @@ void s1ap_remove_enb(s1ap_state_t* state, enb_description_t* enb_ref) {
   hashtable_uint64_ts_destroy(&enb_ref->ue_id_coll);
   hashtable_ts_free(&state->enbs, enb_ref->sctp_assoc_id);
   state->num_enbs--;
+  update_mme_app_stats_connected_enb_sub();
```

Comment:
if this is moved to here, please check if there are other places where s1ap_remove_enb is called as well.